static/panes/editor.js

```diff
@@ -1388,13 +1389,13 @@ Editor.prototype.onCompileResponse = function (compilerId, compiler, result) {
                 if (obj.tag.column) {
                     var span = this.getTokenSpan(obj.tag.line, obj.tag.column);
                     colBegin = obj.tag.column;
-                    if (span.colEnd === obj.tag.column) colEnd = -1;
-                    else colEnd = span.colEnd + 1;
+                    colEnd = span.colEnd;
```

Comment:
thanks for simplifying this - could almost be a ternary now but I think it's worth leaving as-is.